tests/integration_tests/Aspen_HERON_FullTest/gold/new_heron_input.xml

```diff
@@ -49,9 +49,6 @@
           <driver>
             <Function method="capacity">functions</Function>
           </driver>
-          <reference_driver>
-            <fixed_value>10.0</fixed_value>
```

Comment:
so essentially, these were the original reference drivers that instead of their values getting updated, a new node was added and the old one remained?

Reply:
Exactly. The design was that any and all reference_driver, reference_price, and scaling_factor_x nodes were cleared, then all three were added; however, the old nodes were not all deleted.